src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,124 @@
+package wolox.training.models;
+
+
+import javax.persistence.Id;
+
+import javax.persistence.*;
+
+@Entity()
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
When we are working with Hibernate-Postgresql we should use the **SEQUENCE** strategy. This is because the default strategy use a **global** incremental that is shared between all tables.
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```